src/high-level/types/complex-single-float.lisp

```diff
@@ -29,6 +29,37 @@
   (loop :for i :below (size vector1)
         :sum (* (tref vector1 i) (conjugate (tref vector2 i)))))
 
+(defmethod =-lisp ((scalar1 complex) (scalar2 complex) &optional epsilon)
```

Comment:
See comment elsewhere about moving these functions. I think this should be moved to the file I mention elsewhere.